### PR TITLE
Recurring Actions Cucumber tests

### PR DESCRIPTION
**Pull request by parlt91**
## What does this PR change?

Cucumber tests for the recurring actions feature

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: This is just tests

- [x] **DONE**

## Test coverage

- Cucumber tests were added

- [x] **DONE**

## Links

Tracks # https://github.com/SUSE/spacewalk/issues/10940

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"		 
- [ ] Re-run test "java_pgsql_tests"		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"		 
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: testsuite/features/secondary/min_recurring_action.feature [hunk 1]
```diff
@@ -0,0 +1,279 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@scc_credentials
```
Comment (srbarrios) [inline testsuite/features/secondary/min_recurring_action.feature]: I think this test might not need scc credentials

File: testsuite/features/secondary/min_recurring_action.feature [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+Feature: Recurring Actions
+
+  Scenario: Create a minion Recurring Action
+    Given I am authorized as "admin" with password "admin"
+    When I am on the "States" page of this "sle_minion"
```
Comment (srbarrios) [inline testsuite/features/secondary/min_recurring_action.feature]: When -> And
Same in the other scenarios.

Comment (Bischoff) [reply testsuite/features/secondary/min_recurring_action.feature]: I'd say "When" is fine here.

File: testsuite/features/step_definitions/datepicker_steps.rb [hunk 3]
```diff
@@ -87,6 +87,13 @@ def get_future_time(minutes_to_add)
   time.click
 end
 
+Given(/^I pick "([^"]*)" as time from "([^"]*)"$/) do |arg1, arg2|
```
Comment (Bischoff) [inline testsuite/features/step_definitions/datepicker_steps.rb]: "When"

File: testsuite/features/secondary/min_recurring_action.feature (hunk 2 above)
Comment (srbarrios) [reply testsuite/features/secondary/min_recurring_action.feature]: I'm just curious, @Bischoff why? :thinking: 
We agreed that `When` is for actions, like I click, I follow, I pick, I wait, etc.
And almost all our `I am on the` steps, are using `Given`. Why we change of mind here?

**Pull request merged**